Composer/packages/client/src/Onboarding/content.tsx

```diff
@@ -56,12 +56,12 @@ export const stepSets = (projectId: string, rootDialogId: string): IStepSet[] =>
       {
         id: 'userInput',
         navigateTo: `/bot/${projectId}/dialogs/${rootDialogId}?selected=triggers[0]`,
-        targetId: 'navUserInput',
+        targetId: 'navUserinput',
       },
       {
         id: 'botResponses',
         navigateTo: `/bot/${projectId}/dialogs/${rootDialogId}?selected=triggers[0]`,
-        targetId: 'navBotResponses',
+        targetId: 'navBotresponses',
       },
```

Comment:
This seems like a very strange convention, what is the reasoning behind this?

Reply:
These target IDs are made by squishing the spaces out of the names of the pages. Since the in-toolbar name of the "Bot responses" page got a lowercase R (one of the external proofreader's suggestions, along with the other two-word page names), these had to change too in order to keep our Onboarding bubbles intact. This turned out to be the cause of the mysterious e2e test failures.